portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,58 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Diana's Portfolio</h1>
+      <p>I am STEP intern studying <b>Computer Science</b> in
+         the University of Texas at El Paso, currently 
+         working remotely from Mexico. Here's a picture
+         of one of my dogs, he's name is Remmy! </p><br>
+      <img src = "/myImages/remmy.jpeg" alt = "Remmy" width = "200" height = "300" /> <br>
+
+      <h2>My Background</h2>
+      <p> I have the oportunity to study abroad bacause I live in one of the borders 
+        with the US. When I was little my ideas were influeced by a lot of people 
+        due to my country's environment. When I turned 8 years old, I started to 
+        practice Taekwondo, which was a turning point in my life. That environment
+        showed my different perspectives of life, specially when it came to gender 
+        equality. </p><br>
```

Comment:
nit: extra space

Reply:
I see more these spaces, which are not needed in this file

Reply:
Thank you for the review, I'll correct the spaces